aqueduct_client/utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import logging
+
+try:
+    # py3
+    from configparser import ConfigParser
+except ImportError:
+    # py2
+    from ConfigParser import SafeConfigParser as ConfigParser
+
+
+def load_api_key():
+    # check in credentials file
+    path = os.path.expanduser("~/.quantopian/credentials")
+
+    if os.path.isfile(path):
+        # read the file
+        try:
+            parser = ConfigParser()
+            parser.read(path)
+
+            api_key = parser.get("default", "API_KEY")
+            if api_key is not None:
+                return api_key
+        except Exception as e:
+            logging.error(
```

Comment:
This is unlikely to display anything for most users. Is there a reason we don't want to re-raise this exception?